packages/stateless/src/lib/provider/provider.interface.ts

```diff
@@ -32,7 +32,7 @@ export interface IPushOptions {
     tag?: string;
     body?: string;
     icon?: string;
-    badge?: string;
+    badge?: string | number;
```

Comment:
@scopsy @davidsoderberg I have been investigating why this change might be needed and I think we had the `badge` field set to string due FCM options interface: https://github.com/novuhq/novu/blob/b329d9e7c2b115f87f89a9988b7e91a471463a6e/packages/node/src/lib/subscribers/subscriber.interface.ts#L96

But in the README of the provider (https://github.com/novuhq/novu/blob/b53d08c5ca6fa41c8b3ac600594506c965d6b74d/providers/fcm/README.md?plain=1#L20) is mentioned that the use of badge is either 1 or 0, so a number. Should we update this field to `number` directly? Is there any potential trouble that can be caused for that?

Reply:
This seems fine to me but would like to consult @raikasdev that implemented this 😄

Reply:
I don't thing that updating would be any problem as in the transfer of HTTP it will be converted to string anyways. Just need to check for any potential typescript errors during compilation. So we can just do it as `number` @psteinroe

Reply:
If remember correctly, those are just copied from Firebase SDK.